enclave/core/debugmalloc.c

```diff
@@ -72,8 +80,11 @@ struct header
     void* addrs[OE_BACKTRACE_MAX];
     uint64_t num_addrs;
 
+    /* Option if current object is tracked */
+    bool local_tracking;
```

Comment:
Instead of making it a bool, you could change this to:
`int32_t session_number`.

Every time a new tracking session is started, you could increment a global session counter and associate that with the object.

Reply:
Good suggestion, updated, thanks!